tests/foreman/api/test_content_credentials.py

```diff
@@ -0,0 +1,245 @@
+"""Unit tests for the ``gpgkeys`` paths.
+
+:Requirement: ContentCredential
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: ContentCredentials
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+from nailgun import entities
+from requests import HTTPError
+
+from robottelo.constants import VALID_GPG_KEY_BETA_FILE
+from robottelo.constants import VALID_GPG_KEY_FILE
+from robottelo.datafactory import invalid_values_list
+from robottelo.datafactory import parametrized
+from robottelo.datafactory import valid_data_list
+from robottelo.helpers import read_data_file
+
+
+key_content = read_data_file(VALID_GPG_KEY_FILE)
+
+
+@pytest.mark.tier1
+def test_positive_search_in_org(module_org):
```

Comment:
isn't this test pretty much included in every `test_positive_create_with_*` test? IIRC, nailgun used to retrieve the resource as a part of `.create()`.
Also, I can see a potential risk for a race condition - you use a module-based organiation fixture and if any other test that creates GPG keys executes prior this test, your assertions won't pass anymore (or am I missing something?).

Reply:
> isn't this test pretty much included in every `test_positive_create_with_*` test? IIRC, nailgun used to retrieve the resource as a part of `.create()`.

It should.
So really this is just testing the search function

> Also, I can see a potential risk for a race condition - you use a module-based organization fixture and if any other test that creates GPG keys executes prior this test, your assertions won't pass anymore (or am I missing something?).

Yes. If we cannot ensure the test will always run first in the module then I should just delete it. 
I think creating an org just to do this one test is a waste of time.